README.md

```diff
@@ -52,6 +51,74 @@ To reload the extension after some changes, open [`chrome://extensions`](chrome:
 
 To use the local API instead of the prod one, change `ORBIT_ROOT_URL` in `orbit-helper.js` to your ngrok tunnel address.
 
+## Contributing
+
+Pull requests & issues welcome!
+
+### Accessibility
+
+Accessibility is a fundamental part of web design, and something we consider critical to the further development of this extension. Future contirbutions should aim to meet [WCAG2.1 AA](https://www.w3.org/TR/WCAG21/) - if you notice any areas where we're failing, please raise them as issues or submit PRs to address them.
```

Comment:
An [em dash](https://practicaltypography.com/hyphens-and-dashes.htm) would be more accurate here:

> Future contirbutions should aim to meet [WCAG2.1 AA](https://www.w3.org/TR/WCAG21/)—if you notice any areas…

Reply:
TIL the difference between them 🤯 Thanks!!